src/java.base/share/classes/java/io/BufferedInputStream.java

```diff
@@ -243,20 +239,16 @@ public BufferedInputStream(InputStream in, int size) {
         }
         initialSize = size;
         if (getClass() == BufferedInputStream.class) {
-            // use internal lock and lazily create buffer when not subclassed
-            lock = InternalLock.newLockOrNull();
             buf = EMPTY;
```

Comment:
Maybe keep "lazily create buffer when not subclassed" as the comment.

Reply:
See 9417b53.